arkane/statmech.py

```diff
@@ -1529,9 +1529,7 @@ def run(self):
 
 class HinderedRotorClassicalND(Mode):
     """
-    A statistical mechanical model of a 2D-dimensional hindered rotor.
-    Computation of eigenvalues and fourier fitting outsourced to
-    Q2DTor software
+    A statistical mechanical model of a ND-dimensional classical hindered rotor.
```

Comment:
perhaps squash this (50c6055581a1486cbb68c60804d0ac7c17d68c90) commit?

Reply:
fixed